sqlglot/dialects/redshift.py

```diff
@@ -138,6 +138,7 @@ class Tokenizer(Postgres.Tokenizer):
             "TOP": TokenType.TOP,
             "UNLOAD": TokenType.COMMAND,
             "VARBYTE": TokenType.VARBINARY,
+            "ONLY": TokenType.ONLY,
```

Comment:
We can get rid of this, because it's already inherited from Postgres
```suggestion
```

Reply:
True! Done